pkg/config/defs.go

```diff
@@ -23,7 +23,7 @@
 package config
 
 const (
-	KeyActive      string = "active"
-	KeyAlias       string = "aliases"
-	KeyEnvironment string = "environments"
+	KeyAliases            string = "aliases"
+	KeyCurrentEnvironment string = "current-env"
+	KeyEnvironments       string = "envs"
```

Comment:
Maybe call this `env`?

Reply:
since this lists a number of environments and not just a single env with vars, plural should be good?

Reply:
```
current-env: local
envs:
  local:
    namespace: default
  canary:
    namespace: canary
```

Reply:
I'd expect:
```yml
current-env: local
env: # <-- singular
  local:
    namespace: default
  canary:
    namespace: canary
```

Then to use it'd be `tctl config set env.local.namespace default`

Reply:
sure sg